recipes/sphinx-issues/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set version = "0.2.0" %}
+
+package:
+  name: sphinx-issues
+  version: {{ version }}
+
+source:
+  fn: sphinx-issues-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/s/sphinx-issues/sphinx-issues-{{ version }}.tar.gz
+  md5: 6065e4538b8ead02e35a777711eab6f5
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - sphinx
```

Comment:
OK. We really should make the conda skeleton command capable of generating conda-forge ready recipes.